src/Libraries/PythonNodeModels/PythonEngineSelector .cs

```diff
@@ -33,18 +33,26 @@ internal sealed class PythonEngineSelector
         internal static PythonEngineSelector Instance { get { return lazy.Value; } }
 
         // TODO: The following fields might be removed after dynamic loading applied
-        internal static bool IsIronPythonEnabled = false;
-        internal static string IronPythonEvaluatorClass = "IronPythonEvaluator";
-        internal static string IronPythonEvaluationMethod = "EvaluateIronPythonScript";
+        internal bool IsIronPythonEnabled = false;
```

Comment:
These properties do not need to be static but could rather connected with the Singleton itself. So I updated these